src/utils/storageFactory.js

```diff
@@ -50,6 +64,7 @@ export default context => additionalNamespace => {
   const finalNamespace = baseNamespace + (additionalNamespace || "");
   return {
     session: getStorageByType(context, "sessionStorage", finalNamespace),
-    persistent: getStorageByType(context, "localStorage", finalNamespace)
+    persistent: getStorageByType(context, "localStorage", finalNamespace),
+    memory: createMemoryStorage(finalNamespace)
```

Comment:
Volatile keyword here instead of memory, thoughts?

Reply:
@liljenback let's ask the team :)